DEVELOP.rst

```diff
@@ -147,9 +153,26 @@ To see stdout/stderr, use the following command:
 
     $ behave --no-capture
 
+Troubleshooting the integration tests
+-------------------------------------
+
+If you encounter errors when running the tests you might have to edit your
+``pg_hba.conf`` file. The database user has to have permissions to create and
+drop test databases. The default user is ``postgres`` at ``localhost``. Make
+sure the method is set to ``trust``. You might need to set all the ``local``
+connections to ``trust`` as well.
```

Comment:
I think this problem is a bug that needs to be fixed. Tests should be able to run on `local` or `host` connection. I would rather not add this part to docs, but file an issue to fix the problem.